linux_os/guide/system/permissions/restrictions/sysctl_user_max_user_namespaces/rule.yml

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Disable the use of user namespaces'
+
+description: |-
+    {{{ describe_sysctl_option_value(sysctl="user.max_user_namespaces", value="0") }}}
+    When containers are deployed on the machine, the value should be set
+    to large non-zero value.
+
+rationale: |-
+    User namespaces are used primarily for Linux containers. The value 0
+    effectively disallows the use of user namespaces.
+
+severity: low
+
+identifiers:
+    cce@rhel8: 81005-1
+
+references:
+    ospp: FMT_SMF_EXT.1
```

Comment:
What do you think about mapping to NIST 800-53, SC-39? https://nvd.nist.gov/800-53/Rev4/control/SC-39

Reply:
I've added `nist: SC-39` in d518ade748d0ca02854f57d51f6f3174d4394823.